indico/modules/events/timetable/controllers/legacy.py

```diff
@@ -206,3 +212,69 @@ def _process(self):
         with track_time_changes():
             fit_session_block_entry(self.session_block.timetable_entry)
         return jsonify_data(flash=False)
+
+
+class RHLegacyTimetableEntryMove(RHManageTimetableEntryBase):
+    """Moves a TimetableEntry into a Session or top-level timetable"""
+
+    def _process_POST(self):
+        current_day = dateutil.parser.parse(request.form.get('day')).date()
+        return jsonify_template('events/timetable/move_entry.html', event=self.event_new,
+                                top_level_entries=self._get_session_timetable_entries(),
+                                current_day=current_day, timetable_entry=self.entry,
+                                parent_entry=self.entry.parent)
+
+    def _process_PATCH(self):
+        self.serializer = TimetableSerializer(True)
+        with track_time_changes():
+            try:
+                entry_data = self._move_entry(request.json)
+            except Exception, ex:
```

Comment:
Plain `Exception`. I am expecting it when e.g. adding a Contribution into a SessionBlock would make the block span more than 1 day.